resources/pathways_refined.tsv

```diff
@@ -1,49 +1,57 @@
 pathway	reaction	subpathway	signature_gene_name	signature_definition	note
-nitrogen_redox	1+2,1+3+15	nitrate_reduction	napA,narG	K02567,K00370	60% of the KOs for all KOs in reaction
-nitrogen_redox	3+4,3+5+6	denitrification	nirK,nirS	K00368,K15864	70% of the reactions in a pathway need to have some presence
-nitrogen_redox	7	nitrogen_fixation	nifH,vnfH,anfH	K02588,K22899,K02588	Y/N on the signature definition
-nitrogen_redox	8+9	anammox	hzsA	K20934	
-nitrogen_redox	10+11	ammonia_oxidation	amoA	K10944	
-nitrogen_redox	13	nitrite_oxidation	nxrA	K00370	
+nitrogen_redox	1+2,1+3+15	nitrate_reduction			60% of the KOs for all KOs in reaction
+nitrogen_redox	3+4,3+5+6	denitrification			70% of the reactions in a pathway need to have some presence
+nitrogen_redox	7	nitrogen_fixation			Y/N on the signature definition
+nitrogen_redox	8+9	anammox			
+nitrogen_redox	10+11	ammonia_oxidation			
+nitrogen_redox	13	nitrite_oxidation			
 nitrogen_redox	14	nitric_oxide_oxidation			
-nitrogen_redox	10+11+13	comammox	amoA+nxrA	K10944+K00370	
+nitrogen_redox	10+11+13	comammox	(pmoC,pmoA,pmoB,hao)+(nxrA,nxrB)	(K10946,K10944,K10945,K10535)+(K00370,K00371)	
```

Comment:
This signature definition has brackets. Will this still work?

Reply:
Yes it should, I can't recall if I tested it though.